packages/cloudvision-connector/types/params.ts

```diff
@@ -118,6 +122,10 @@ export interface AppParams {
   types?: DatasetType[];
 }
 
+export interface ConfigParams {
```

Comment:
Rename appparams type. We shouldn't be creating a new one for each type.

Reply:
Done. Renamed `AppsParams` to `DatasetsParams` (not sure if `GetDatasetsParams` is a better name).